lib/Installation/Partitioner/LibstorageNG/v4_3/CreatePartitionTablePage.pm

```diff
@@ -36,4 +37,9 @@ sub press_next {
     return $self->{btn_next}->click();
 }
 
+sub select_partition_table_type {
```

Comment:
Same here, feels like this should be `select_msdos_partition_table_type` as it doesn't allow selecting gpt. For that we could add a parameter.

Reply:
previously, we were using args->{table_type} form test data here, but I see here we put the desired value in init() . I don't know what's the reason, but I guess it's better to keep it consistent ?